apps/web/components/ui/HeroCarousel/HeroCarousel.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <Swiper
+      :modules="enableModules ? [Pagination, Navigation] : []"
+      :slides-per-view="1"
+      :navigation="enableModules ? true : false"
+      :loop="true"
+      :autoplay="{
+        delay: 3000,
+        disableOnInteraction: false,
+      }"
+      pagination
+      @slide-change="onSlideChange"
+      wrapper-class="md:px-7 lg:px-11"
+    >
+      <SwiperSlide v-for="(heroItem, index) in heroProps" :key="index" style="height: 100%">
```

Comment:
**question**: Is there a reason to use `style="height: 100%"` instead of `class="h-100"`? If at all possible, we should prefer Tailwind.

Reply:
I need inline styling because swiper out of the box classes will take priority over tailwind.

Reply:
Nvm,this seems to work if I add it via code. Strange the effect did not take place via inspector.